submissions/MishaDerkach/html-css-popup/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="header">
+        <nav class="nav">
```

Comment:
I think you have enough space to put it in one line on mobile)
![image](https://user-images.githubusercontent.com/16196199/187023784-466e3b05-6839-437f-b81e-49025e4c08ec.png)